### ISSUE TITLE
Touch feedback on headers possible

### ISSUE THREAD

**Issue by MFlisar** [timehop/sticky-headers-recyclerview#55]
Is that possible? The headers don't get the focus, so buttons and similiar are not possible on decorators.

But is it possible to at least give some touch feedback?

**Comment by forcewill** [comment]
I actually contributed with sound click on header clicking, don't you get a sound when clicking on the headers ?

**Comment by MFlisar** [comment]
No, I don't... Actually, I would like to have some visual feedback... sound or vibration wouldn't be a problem...

**Comment by jacobtabak** [comment]
It's not possible without reimplementing all of those touch states
manually. The headers are decorations that are drawn on screen, not actual
views with touch states. Reasonable contributions will be considered.
